knowledge_base/job_with_custom_sys_path/src/print.ipynb

```diff
@@ -0,0 +1,49 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The below cell retrieves the path to this bundle's deployment file path,\n",
+    "and adds it to the Python path."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "from databricks.sdk.runtime import dbutils\n",
+    "bundle_file_path = dbutils.widgets.get(\"bundle_file_path\")\n",
+    "sys.path.append(bundle_file_path)"
```

Comment:
I'm curious if this solves the customer problem. But there's another approach here where you could support interactive execution as well: you could have a `%run ./add-sys-path` here. The `add-sys-path` would then be a notebook that does something like https://github.com/databricks/bundle-examples/blob/main/default_python/fixtures/.gitkeep#L11 to add its containing folder to the sys path.